components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2958,9 +2958,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
 
     public async getGitpodTokens(ctx: TraceContext): Promise<GitpodToken[]> {
         const user = await this.checkAndBlockUser("getGitpodTokens");
-        const res = (await this.userDB.findAllGitpodTokensOfUser(user.id)).filter((v) => !v.deleted);
-        await Promise.all(res.map((tkn) => this.guardAccess({ kind: "gitpodToken", subject: tkn }, "get")));
```

Comment:
keep the guardAccess here. don'T move it into the services. `guardAccess´ is the old permission system we want to remove it once we have migrated all calls to use the new `authorizer` checks.

Reply:
it's tricky because check is done at the end before writing to db, I talked about it yesterdat with Gero, if not I'll have to duplicate code